### PR TITLE
I found the fixme package :tiny_potato:

### PR DESCRIPTION
**Pull request by Sollace**
// TODO:

### CODE REVIEW

File: mappings/net/minecraft/class_4922.mapping [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+	METHOD <init> (Lnet/minecraft/class_2248;)V
+		ARG 1 block
+	METHOD method_25758 create (Lnet/minecraft/class_2248;)Lnet/minecraft/class_4922;
+		ARG 0 block
+	CLASS class_4923
+		FIELD field_22857 when Lnet/minecraft/class_4918;
+		METHOD <init> (Lnet/minecraft/class_4918;Ljava/util/List;)V
+			ARG 1 when
+	CLASS class_4924
+		METHOD get getParts ()Ljava/lang/Object;
```
Comment (liach) [inline mappings/net/minecraft/class_4922.mapping]: 🤔

Comment (Sollace) [reply mappings/net/minecraft/class_4922.mapping]: `ARG 1 when`

It's a datafixer and Mojang mentions missing parts in conjunction with this method.

Comment (liach) [reply mappings/net/minecraft/class_4922.mapping]: I am :thinking: on why you renamed `Supplier.get` implementation

Comment (Sollace) [reply mappings/net/minecraft/class_4922.mapping]: Because it's not a `Supplier.get` implementation?

Comment (Sollace) [reply mappings/net/minecraft/class_4922.mapping]: Welp. Looks like it was a get. Fixed now.

File: mappings/com/mojang/realmsclient/dto/ValueObject.mapping [hunk 2]
```diff
@@ -1,3 +1,5 @@
 CLASS net/minecraft/class_4352 com/mojang/realmsclient/dto/ValueObject
-	METHOD method_25094 (Ljava/lang/reflect/Field;)Z
+	METHOD method_25093 getname (Ljava/lang/reflect/Field;)Ljava/lang/String;
```
Comment (YanisBft) [inline mappings/com/mojang/realmsclient/dto/ValueObject.mapping]: `getName`

File: mappings/com/mojang/realmsclient/network/Backup.mapping [hunk 3]
```diff
@@ -0,0 +1,14 @@
+CLASS net/minecraft/class_4867 com/mojang/realmsclient/network/Backup
+	FIELD field_22569 backupId Ljava/lang/String;
+	FIELD field_22570 lastModifiedDate Ljava/util/Date;
+	FIELD field_22571 size J
+	FIELD field_22572 metadata Ljava/util/Map;
+	FIELD field_22574 LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD field_22575 uploadedVersion Z
+	METHOD method_25032 getuploadedVersion ()Z
```
Comment (YanisBft) [inline mappings/com/mojang/realmsclient/network/Backup.mapping]: `getUploadedVersion`

File: mappings/com/mojang/realmsclient/network/Backup.mapping [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+	METHOD method_25033 parse (Lcom/google/gson/JsonElement;)Lnet/minecraft/class_4867;
+		ARG 0 node
+	METHOD method_25034 parseMetadata (Ljava/lang/String;)Ljava/lang/String;
+		ARG 0 key
+	METHOD method_25035 etUploadedVersion (Z)V
```
Comment (YanisBft) [inline mappings/com/mojang/realmsclient/network/Backup.mapping]: `setUploadedVersion`

File: mappings/com/mojang/realmsclient/network/McoServer.mapping [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+	METHOD method_25069 emptySlots ()Ljava/util/Map;
+	METHOD method_25070 getOrNormal (Ljava/lang/String;)Lnet/minecraft/class_4877$class_4321;
+		ARG 0 state
+	CLASS class_4319 McoServerComparator
+		FIELD field_19432 refOwner Ljava/lang/String;
+		METHOD <init> (Ljava/lang/String;)V
+			ARG 1 owner
+		METHOD compare (Ljava/lang/Object;Ljava/lang/Object;)I
+			ARG 1 one
+			ARG 2 two
```
Comment (YanisBft) [inline mappings/com/mojang/realmsclient/network/McoServer.mapping]: `first` and `second` instead?

File: mappings/com/mojang/realmsclient/network/PingResults.mapping [hunk 6]
```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/class_4873 com/mojang/realmsclient/network/PingResults
+	FIELD field_22588 pinResults Ljava/util/List;
+	FIELD field_22589 worldids Ljava/util/List;
```
Comment (YanisBft) [inline mappings/com/mojang/realmsclient/network/PingResults.mapping]: `pingResults`
`worldIds`

File: mappings/com/mojang/realmsclient/network/Player.mapping [hunk 7]
```diff
@@ -0,0 +1,21 @@
+CLASS net/minecraft/class_4874 com/mojang/realmsclient/network/Player
+	FIELD field_22590 name Ljava/lang/String;
+	FIELD field_22591 uuid Ljava/lang/String;
+	FIELD field_22592 operator Z
+	FIELD field_22593 accepted Z
+	FIELD field_22594 online Z
+	METHOD method_25042 getname ()Ljava/lang/String;
```
Comment (YanisBft) [inline mappings/com/mojang/realmsclient/network/Player.mapping]: `getName`

File: mappings/com/mojang/realmsclient/network/Slot.mapping [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+	FIELD field_22642 worldTemplateId J
+	FIELD field_22643 worldTemplateImage Ljava/lang/String;
+	FIELD field_22644 adventureMap Z
+	FIELD field_22645 empty Z
+	FIELD field_22646 DEFAULT_WORLD_TEMPLATE_IMAGE Ljava/lang/String;
+	METHOD <init> (Ljava/lang/Boolean;Ljava/lang/Boolean;Ljava/lang/Boolean;Ljava/lang/Boolean;Ljava/lang/Integer;Ljava/lang/Boolean;Ljava/lang/Integer;Ljava/lang/Integer;Ljava/lang/Boolean;Ljava/lang/String;)V
+		ARG 1 pvp
+		ARG 2 spawnAnimals
+		ARG 3 spawnMonsters
+		ARG 4 spawnNPCs
```
Comment (YanisBft) [inline mappings/com/mojang/realmsclient/network/Slot.mapping]: `spawnNpcs` to be consistent with the field above

File: mappings/com/mojang/realmsclient/network/UploadInfo.mapping [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+	FIELD field_22657 LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD field_22658 worldClosed Z
+	FIELD field_22659 token Ljava/lang/String;
+	FIELD field_22660 uploadEndpoint Ljava/lang/String;
+	FIELD field_22661 port I
+	METHOD method_25087 getToken ()Ljava/lang/String;
+	METHOD method_25088 parse (Ljava/lang/String;)Lnet/minecraft/class_4888;
+		ARG 0 json
+	METHOD method_25089 getEndpoint ()Ljava/lang/String;
+	METHOD method_25090 ssetToken (Ljava/lang/String;)V
```
Comment (YanisBft) [inline mappings/com/mojang/realmsclient/network/UploadInfo.mapping]: `setToken`

File: mappings/com/mojang/realmsclient/network/WorldDownload.mapping [hunk 10]
```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/class_4889 com/mojang/realmsclient/network/WorldDownload
+	FIELD field_22662 downloadLink Ljava/lang/String;
+	FIELD field_22663 reourcepackUrl Ljava/lang/String;
+	FIELD field_22664 reourcepackHah Ljava/lang/String;
```
Comment (YanisBft) [inline mappings/com/mojang/realmsclient/network/WorldDownload.mapping]: `resourcePackUrl` and I guess `resourcePackHash`

**Review by YanisBft — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/fixme/AbstractCraftingContainer.mapping [hunk 11]
```diff
@@ -0,0 +1,18 @@
...
+	FIELD field_22482 player Lnet/minecraft/class_1657;
+	METHOD <init> (Lnet/minecraft/class_3917;ILnet/minecraft/class_1661;Lnet/minecraft/class_3914;)V
+		ARG 1 type
+		ARG 2 id
+		ARG 3 playerInventory
+		ARG 4 context
+	METHOD method_24923 onCrafting (Lnet/minecraft/class_1657;Lnet/minecraft/class_1799;)Lnet/minecraft/class_1799;
+		ARG 1 player
+		ARG 2 stack
+	METHOD method_24925 iCraftingTable (Lnet/minecraft/class_2680;)Z
```
Comment (YanisBft) [inline mappings/net/minecraft/fixme/AbstractCraftingContainer.mapping]: Forgot this one, `isCraftingTable`

File: mappings/net/minecraft/fixme/GeneratedItemModels.mapping [hunk 12]
```diff
@@ -0,0 +1,10 @@
+CLASS net/minecraft/class_4915 net/minecraft/fixme/GeneratedItemModels
+	FIELD field_22844 deSerializer Ljava/util/function/BiConsumer;
```
Comment (Earthcomputer) [inline mappings/net/minecraft/fixme/GeneratedItemModels.mapping]: "deserializer" is one word

**Review by Earthcomputer — CHANGES_REQUESTED** [review]

**Comment by liach** [comment]
Need to update your pr as container is gone

**Comment by Sollace** [comment]
> Need to update your pr as container is gone

:crab_dance:

**Comment by Sollace** [comment]
@liach @Yanis48 I'll fix all the errors. Most of these are due to my keyboard being f***cked to hell.

File: mappings/com/mojang/realmsclient/network/Slot.mapping (hunk 8 above)
Comment (Sollace) [reply mappings/com/mojang/realmsclient/network/Slot.mapping]: if (!this.spawnNPCs) {
          jsonObject2.addProperty("spawnNPCs", this.spawnNPCs);
    }

**Comment by Sollace** [comment]
New PR for the realms classes: #1170

**Comment by Sollace** [comment]
New PR for just the util package changes: #1171

**Comment by Sollace** [comment]
Closing this as there is a new PR: #1172

**Pull request closed without merging**